doc-Installing_on_OpenShift_Container_Platform/topics/installation.adoc

```diff
@@ -26,7 +26,18 @@ To successfully deploy a {product-title} appliance on OpenShift Container Platfo
 OpenShift Container Platform 3.5 is required for this installation. Red Hat has not tested this procedure with earlier versions of OpenShift Container Platform.
 ====
 
-The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. OpenShift Container Platform 3.5 includes these files by default.
+The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. 
+
+These files create the OpenShift pods comprising the appliance, and are available from the Red Hat Container Catalog. To download the files to your system, follow the OpenShift instructions on the *Get this image* page for each of the following images:
```

Comment:
Thanks for the suggestion, Suyog -- I think it's a good idea to be a little more clear. I've edited this sentence to better reflect the procedure (and language) on the Container Catalog pages. Please let me know your thoughts. :)